src/base.ts

```diff
@@ -246,12 +259,12 @@ Please check your network connection.`);
     }
 
     if (!config['api-token'] || !config.region) {
-      const { api_token, region } = config.account
+      const { api_token } = config.account
         ? await this.getAccount(config.account)
         : await this.getCurrentAccount();
 
       config['api-token'] = config['api-token'] || api_token;
-      config.region = config.region || region;
+      // config.region = config.region || region;
     }
```

Comment:
Don't forget that all accounts have a `region` field in the global configuration file `liara-auth.json`

I believe we shouldn't modify this global config file. To eliminate the prompt asking for the region, we can simply set a fallback region to `"IRAN"` and never prompt the user for a region.

Additionally, we should only remove the `--region` flag for simplicity.

For backward compatibility, in my opinion, we should automatically set the region to "IRAN". This ensures nothing unexpected happens in the CLI and keeps the region field hidden from the user.

This approach will streamline the process and improve the user experience.

PING @mhemrg

Reply:
> I believe we shouldn't modify this global config file. To eliminate the prompt asking for the region, we can simply set a fallback region to `"IRAN"` and never prompt the user for a region.

Fallback region has already been implemented. Please search for `FALLBACK_REGION` throughout the repo.